### PR TITLE
Corrected inline syntax in documentation

### PR DESCRIPTION
**Pull request by Denton-L**

### CODE REVIEW

File: docs/frequently-asked-questions.rst [hunk 1]
```diff
@@ -71,8 +71,8 @@ several blockchain explorers.
 Contracts on the blockchain should have their original source
 code published if they are to be used by third parties.
 
-Does selfdestruct() free up space in the blockchain?
-====================================================
+Does ``selfdestruct()`` free up space in the blockchain?
+============================================================
```
Comment (chriseth) [inline docs/frequently-asked-questions.rst]: This is too long now.

File: docs/frequently-asked-questions.rst [hunk 2]
```diff
@@ -89,91 +89,91 @@ If you want to deactivate your contracts, it is preferable to **disable** them b
...
 
-The trick is to create the contract with `{from:someaddress, value: web3.toWei(3,"ether")...}`
+The trick is to create the contract with ``{from:someaddress, value: web3.toWei(3,"ether")...}``
 
 See `endowment_retriever.sol <https://github.com/fivedogit/solidity-baby-steps/blob/master/contracts/30_endowment_retriever.sol>`_.
 
-Use a non-constant function (req sendTransaction) to increment a variable in a contract
-=======================================================================================
+Use a non-constant function (req ``sendTransaction``) to increment a variable in a contract
+===============================================================================================
```
Comment (chriseth) [inline docs/frequently-asked-questions.rst]: Also too long here.

File: docs/frequently-asked-questions.rst [hunk 3]
```diff
@@ -89,91 +89,91 @@ If you want to deactivate your contracts, it is preferable to **disable** them b
...
+Long answer: ``this`` is a variable representing the current contract.
 Its type is the type of the contract. Since any contract type basically inherits from the
-`address` type, `this` is always convertible to `address` and in this case contains
+``address`` type, ``this`` is always convertible to ``address`` and in this case contains
 its own address.
 
-What is the difference between a function marked constant and one that is not?
-==============================================================================
+What is the difference between a function marked ``constant`` and one that is not?
+======================================================================================
```
Comment (chriseth) [inline docs/frequently-asked-questions.rst]: Again too long, also others below.

File: docs/frequently-asked-questions.rst [hunk 4]
```diff
@@ -368,16 +368,16 @@ Examples::
...
+=====================================
 
 Modifiers are a way to prepend or append code to a function in order
 to add guards, initialisation or cleanup functionality in a concise way.
 
 For examples, see the `features.sol <https://github.com/ethereum/dapp-bin/blob/master/library/features.sol>`_.
 
-How do structs work?
-====================
+How do ``struct``'s work?
```
Comment (chriseth) [inline docs/frequently-asked-questions.rst]: I think this should be kept as it is - it is not code.

File: docs/frequently-asked-questions.rst [hunk 5]
```diff
@@ -402,14 +402,14 @@ Solidity is character set agnostic concerning strings in the source code, althou
 utf-8 is recommended. Identifiers (variables, functions, ...) can only use
 ASCII.
 
-What are some examples of basic string manipulation (substring, indexOf, charAt, etc)?
-======================================================================================
+What are some examples of basic string manipulation (``substring``, ``indexOf``, ``charAt``, etc)?
+==============================================================================================================
 
 There are some string utility functions at `stringUtils.sol <https://github.com/ethereum/dapp-bin/blob/master/library/stringUtils.sol>`_
```
Comment (chriseth) [inline docs/frequently-asked-questions.rst:408]: We could include @arachnid's new library here.

File: docs/frequently-asked-questions.rst [hunk 6]
```diff
@@ -839,12 +839,12 @@ https://github.com/ethereum/wiki/wiki/Subtleties
...
-=====================
+How do I use ``.send()``?
+=============================
 
-If you want to send 20 Ether from a contract to the address `x`, you use `x.send(20 ether);`.
-Here, `x` can be a plain address or a contract. If the contract already explicitly defines
-a function `send` (and thus overwrites the special function), you can use `address(x).send(20 ether);`.
+If you want to send 20 Ether from a contract to the address ``x``, you use ``x.send(20 ether);``.
+Here, ``x`` can be a plain address or a contract. If the contract already explicitly defines
+a function ``send`` (and thus overwrites the special function), you can use ``address(x).send(20 ether);``.
```
Comment (chriseth) [inline docs/frequently-asked-questions.rst:847]: We should warn users that they always have to check the return value of send.

**Comment by Denton-L** [comment]
Rebased to fix merge conflicts.

**Comment by Denton-L** [comment]
Rebased again to fix merge conflicts.

**Pull request merged**